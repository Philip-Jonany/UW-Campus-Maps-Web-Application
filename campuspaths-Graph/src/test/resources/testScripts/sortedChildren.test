CreateGraph A

AddNode A n0
AddNode A n1
AddNode A n2
AddNode A n3
AddNode A n4

AddEdge A n0 n4 e1
AddEdge A n0 n1 e2
AddEdge A n0 n1 e1
AddEdge A n0 n2 e1
AddEdge A n0 n2 e2
AddEdge A n0 n3 e1

ListChildren A n0