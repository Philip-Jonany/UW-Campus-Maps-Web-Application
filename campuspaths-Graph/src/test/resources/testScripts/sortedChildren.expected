created graph A

added node n0 to A
added node n1 to A
added node n2 to A
added node n3 to A
added node n4 to A

added edge e1 from n0 to n4 in A
added edge e2 from n0 to n1 in A
added edge e1 from n0 to n1 in A
added edge e1 from n0 to n2 in A
added edge e2 from n0 to n2 in A
added edge e1 from n0 to n3 in A

the children of n0 in A are: n1(e1) n1(e2) n2(e1) n2(e2) n3(e1) n4(e1)